diff --git a/Dockerfile-Ubuntu-22.04 b/Dockerfile-Ubuntu-22.04
index ff03738..391da07 100644
--- a/Dockerfile-Ubuntu-22.04
+++ b/Dockerfile-Ubuntu-22.04
@@ -26,6 +26,10 @@ RUN rm /bin/sh && ln -s bash /bin/sh
 ADD https://storage.googleapis.com/git-repo-downloads/repo /usr/local/bin/
 RUN chmod 755 /usr/local/bin/repo
 
+# Correct Python path
+RUN update-alternatives --install /usr/bin/python python /usr/bin/python3.10 1
+RUN update-alternatives --config python
+
 # Add your user to sudoers to be able to install other packages in the container.
 ARG USER
 RUN echo "${USER} ALL=(ALL) NOPASSWD: ALL" > /etc/sudoers.d/${USER} && \
diff --git a/docker-build.sh b/docker-build.sh
index 1e2afc9..0fa50d6 100755
--- a/docker-build.sh
+++ b/docker-build.sh
@@ -23,6 +23,7 @@ if [ $# -ne 1 ]
                      --build-arg "USER=$(whoami)" \
                      --build-arg "host_uid=$(id -u)" \
                      --build-arg "host_gid=$(id -g)" \
+                     --network host \
                      -f $1 \
                      .
 fi
